ospfd/ospf_interface.c

```diff
@@ -845,18 +845,20 @@ int ospf_if_down(struct ospf_interface *oi)
 	/* Shutdown packet reception and sending */
 	ospf_if_stream_unset(oi);
 
-	for (rn = route_top(ospf->new_table); rn; rn = route_next(rn)) {
-		or = rn->info;
+	if (ospf->new_table) {
+		for (rn = route_top(ospf->new_table); rn; rn = route_next(rn)) {
+			or = rn->info;
 
-		if (!or)
-			continue;
+			if (! or)
```

Comment:
did a clang-format for the range, so I guess this is OK according to the coding standard.

Reply:
Can you drop a whitespace after `!`?

Reply:
I'll try to find what's wrong in .clang-format

Reply:
Must be some bug in clang-format as changing the variable name makes clang-format format the lines OK.

Reply:
or is seen as a keyword.  @ton31337 is correct in what it should look like